pkgs/development/libraries/libva/default.nix

```diff
@@ -66,4 +66,8 @@ stdenv.mkDerivation (finalAttrs: {
       lib.systems.inspect.platformPatterns.isStatic
     ];
   };
+} // lib.optionalAttrs (stdenv.cc.bintools.isLLVM && lib.versionAtLeast stdenv.cc.bintools.version "17") {
```

Comment:
Please use env. That kind of attrset merging is making derivations really unreadable.

Reply:
#335255